Makefile

```diff
@@ -1,10 +1,21 @@
+local-unit:
+	pytest test/unit
+	
+local-integ:
+	pytest test/integration/blackbox_test.py test/integration/dryrun_mixin_docs_test.py
+	# NOT WORKING: pytest test/integration/integration_test.py
+
 UNITS = "@unit.abijson or @unit.algod or @unit.applications or @unit.atomic_transaction_composer or @unit.dryrun or @unit.feetest or @unit.indexer or @unit.indexer.logs or @unit.offline or @unit.rekey or @unit.transactions.keyreg or @unit.responses or @unit.responses.231 or @unit.tealsign or @unit.transactions or @unit.transactions.payment or @unit.responses.unlimited_assets or @unit.indexer.ledger_refactoring or @unit.algod.ledger_refactoring"
-unit:
+cuke-unit:
 	behave --tags=$(UNITS) test -f progress2
 
-INTEGRATIONS = "@abi or @algod or @applications or @applications.verified or @assets or @auction or @c2c or @compile or @dryrun or @dryrun.testing or @indexer or @indexer.231 or @indexer.applications or @kmd or @rekey or @send.keyregtxn or @send"
-integration:
-	behave --tags=$(INTEGRATIONS) test -f progress2
+INTEGS = "@abi or @algod or @applications or @applications.verified or @assets or @auction or @c2c or @compile or @dryrun or @dryrun.testing or @indexer or @indexer.231 or @indexer.applications or @kmd or @rekey or @send.keyregtxn or @send"
+cuke-integ:
```

Comment:
nit: can we use `cucumber-unit` and `cucumber-integration` instead? It's more specific, and there's no need to be brief because the common use case will be to run `make unit` or `make integration` instead of invoking these individually.

Also, if `make unit` runs `local-unit` now, it's not strictly necessary to have a step for `make local-unit` in `.circleci/config.yml`, since the unit tests will get ran by the algorand-sdk-testing harness